### PR TITLE
Updated jquery.dataTables for 1.10.7.

### PR DESCRIPTION
**Pull request by CNBoland**
Version release notes: https://cdn.datatables.net/1.10.7/
- Added any()
- Added i18n()

Note: selector-modifier support currently exists with interface ObjectSelectorModifier.

### CODE REVIEW

File: jquery.dataTables/jquery.dataTables-tests.ts [hunk 1]
```diff
@@ -311,6 +311,9 @@ $(document).ready(function () {
 
     var initSettings = dt.init();
 
+    var i18n = dt.i18n('buttons.copy', 'Copy to clipboard');
```
Comment (TonyPythoneer) [inline jquery.dataTables/jquery.dataTables-tests.ts]: I think `var i18n: string` is better.

File: jquery.dataTables/jquery.dataTables-tests.ts [hunk 2]
```diff
@@ -901,5 +904,7 @@ $(document).ready(function () {
 
     //#region "Methods-Util"
 
+    var util_1 = dt.any();
```
Comment (TonyPythoneer) [inline jquery.dataTables/jquery.dataTables-tests.ts]: I think `var util_1: boolean` is better.

**Review by TonyPythoneer — COMMENTED** [review]
The test file is to test type, not function. You can refer my comment. 😄

File: jquery.dataTables/jquery.dataTables-tests.ts (hunk 1 above)
Comment (CNBoland) [inline jquery.dataTables/jquery.dataTables-tests.ts]: Done.

File: jquery.dataTables/jquery.dataTables-tests.ts (hunk 2 above)
Comment (CNBoland) [inline jquery.dataTables/jquery.dataTables-tests.ts]: Done.

File: jquery.dataTables/jquery.dataTables.d.ts [hunk 3]
```diff
@@ -189,6 +189,17 @@ declare namespace DataTables {
...
         /*
+        * Look up a language token that was defined in the DataTables' language initialisation object.
+        *
+        * @param token The language token to lookup from the language object.
+        * @param def The default value to use if the DataTables initialisation has not specified a value.
+        * @param numeric If handling numeric output, the number to be presented should be given in this parameter. If not numeric operator is required (for example button label text) this parameter is not required.
+        *
+        * @returns Resulting internationalised string.
+        */
+        i18n(token: string, def: Object | string, numeric?: number): string;
```
Comment (vvakame) [inline jquery.dataTables/jquery.dataTables.d.ts]: please use `any` instead of `Object`.

Comment (CNBoland) [inline jquery.dataTables/jquery.dataTables.d.ts]: I can do that, but to be clear, the documentation defines the 'def' parameter as object or string: https://datatables.net/reference/api/i18n(). What do you want me to do?

Comment (vvakame) [inline jquery.dataTables/jquery.dataTables.d.ts]: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html
> If you’re tempted to use the type Object, consider using any instead. There is currently no way in TypeScript to specify an object that is “not a primitive”.

Comment (CNBoland) [inline jquery.dataTables/jquery.dataTables.d.ts]: Done

**Pull request merged**